File: dart-client/lib/model/edge_module_added_object.dart
part of swagger.api;

class EdgeModuleAddedObject {
  
  String id = null;
  

  String modulename = null;
  
  EdgeModuleAddedObject();

  @override
  String toString() {
    return 'EdgeModuleAddedObject[id=$id, modulename=$modulename, ]';
  }

  EdgeModuleAddedObject.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    id =
        json['id']
    ;
    modulename =
        json['modulename']
    ;
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'modulename': modulename
     };
  }

  static List<EdgeModuleAddedObject> listFromJson(List<dynamic> json) {
    return json == null ? new List<EdgeModuleAddedObject>() : json.map((value) => new EdgeModuleAddedObject.fromJson(value)).toList();
  }

  static Map<String, EdgeModuleAddedObject> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, EdgeModuleAddedObject>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new EdgeModuleAddedObject.fromJson(value));
    }
    return map;
  }
}

File: csharp-dotnet2-client/src/main/CsharpDotNet2/IO/Swagger/Model/EdgeModuleAddedObject.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class EdgeModuleAddedObject {
    /// <summary>
    /// Gets or Sets Id
    /// </summary>
    [DataMember(Name="_id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "_id")]
    public string Id { get; set; }

    /// <summary>
    /// Gets or Sets Modulename
    /// </summary>
    [DataMember(Name="modulename", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "modulename")]
    public string Modulename { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class EdgeModuleAddedObject {\n");
      sb.Append("  Id: ").Append(Id).Append("\n");
      sb.Append("  Modulename: ").Append(Modulename).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
